app/javascript/dashboard/components/helpCenter/Sidebar/SidebarSearch.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <div class="search-input--wrap">
+    <div class="search-icon--wrap">
+      <fluent-icon icon="search" class="search-icon" />
+    </div>
+    <input
+      v-model="searchValue"
+      class="search-input"
+      placeholder="Search for articles"
```

Comment:
i18n.

Also, why does this component exist? 
Since it is not emitting anything.